applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -355,16 +384,22 @@ int bt_le_audio_tx_init(void)
 		return -EALREADY;
 	}
 
-	for (int i = 0; i < CONFIG_BT_ISO_MAX_CHAN; i++) {
-		tx_info_arr[i].iso_tx_pool = iso_tx_pools[i];
-		tx_info_arr[i].hci_wrn_printed = false;
-		tx_info_arr[i].iso_conn_handle = HANDLE_INVALID;
-		tx_info_arr[i].iso_tx.ts = 0;
-		tx_info_arr[i].iso_tx.offset = 0;
-		tx_info_arr[i].iso_tx.seq_num = 0;
-		tx_info_arr[i].iso_tx_readback.ts = 0;
-		tx_info_arr[i].iso_tx_readback.offset = 0;
-		tx_info_arr[i].iso_tx_readback.seq_num = 0;
+	uint8_t tx_pool_index = 0;
+
+	for (int i = 0; i < GROUP_MAX; i++) {
+		for (int j = 0; j < SUBGROUP_MAX; j++) {
+			for (int k = 0; k < TX_STREAMS_MAX; k++) {
```

Comment:
Discuss naming. group and subgroup makes sense, but TX_STREAMS sounds a tad strange.

Reply:
Suggestions? Tx streams are what they are though :P

Reply:
```suggestion
			for (int k = 0; k < STREAMS_MAX; k++) {
```